src/components/course-settings/period-roster.cjsx

```diff
@@ -25,6 +25,7 @@ module.exports = React.createClass
 
   render: ->
     students = _.sortBy(RosterStore.getStudentsForPeriod(@props.courseId, @props.period.id), 'last_name')
+    students = _.filter(students, (student) -> student.active)
```

Comment:
Since you're just testing a property, you could use [where](http://underscorejs.org/#where)

`students = _.where(students, active: true)`